.github/ISSUE_TEMPLATE/release.yml

```diff
@@ -99,17 +98,31 @@ body:
       label: Hotfix release checklist
       description: Complete these items in sequence as the `Hotfix` release progresses
       options:
-        - label: Make the hotfix in a branch from `prod`
-        - label: Bump the application version
+        - label: Create a hotfix branch from the latest release tag on `main` (`git checkout -b hotfix YYYY.0M.R`)
+        - label: Commit the fixes to the hotfix branch
         - label: Ensure `prod` secrets are up to date
-        - label: Open a PR from the hotfix branch into `prod`, merge
+        - label: Create an annotated tag on the hotfix branch (`git tag -a YYYY.0M.R`)
+        - label: Push the annotated tag (`git push origin YYYY.0M.R`)
         - label: Smoke Test the fix in prod
         - label: Confirm acceptance of Smoke Tests by adding a comment to this issue
-        - label: Tag the release on `prod`, push the tag to GitHub (see [docs](https://docs.calitp.org/benefits/deployment/release/#5-tag-the-release) for commands)
-        - label: Create a release in GitHub for the tag, generating release notes
         - label: Edit release notes to link back to this Release process issue
         - label: Edit release notes with additional context, images, animations, etc. as-needed
-        - label: Create a branch off `main`
-        - label: Open a PR from `prod` into that branch, merge
-        - label: (optional) Adapt hotfix to current state of `main` in that same branch
-        - label: Merge the PR into `main`
+        - label: (optional) Open a PR from the hotfix branch to the current state of `main`
+        - label: (optional) Merge the PR into `main`
```

Comment:
I think in general, these steps should not be optional, i.e. we would want the fix to be re-integrated into `main` to ensure it's in future releases. 

Maybe there's an edge case where the bug doesn't exist in `main` because it was fixed in in-progress work and so it's not necessary to merge the hotfix branch, but I think this would be less likely to happen.

Reply:
```suggestion
        - label: Open a PR from the hotfix branch to the current state of `main`
        - label: Merge the PR into `main`
```

Reply:
That makes sense @angela-tran ! I made this change in f27904e30c4bb1c4ca1240de8c2575f542caeb70